Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -333,9 +344,9 @@ int main(int argc, char** argv)
         frequency_menu.add_action(*action);
     };
 
-    make_frequency_action(1);
-    make_frequency_action(3, true);
-    make_frequency_action(5);
+    make_frequency_action(1, frequency == 1);
+    make_frequency_action(3, frequency == 3);
+    make_frequency_action(5, frequency == 5);
```

Comment:
I'm thinking what happens if someone manually changes the config to a value that's not in this list. None of the items would be checked by default which I guess is fine...

Reply:
Right after reading it I made so that if it's not 1 or 3 or 5 it will get reverted to 3, so at least one of these will be checked.